lib/mongoid/tasks/database.rb

```diff
@@ -26,6 +26,9 @@ def create_collections(models = ::Mongoid.models, force: false)
           else
             logger.info("MONGOID: collection options ignored on: #{model}, please define in the root model.")
           end
+        rescue Exception
+          puts "error while creating collection for #{model}"
```

Comment:
We use `logger` above, should we use it here, too?

Reply:
For whatever reason, the logger output doesn't display in CI, which is where I had to look to troubleshoot this initially. I can dig in and see why the logger output isn't being displayed; if I can figure that out, then I'll switch the `puts` to the logger here.